lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -69,7 +70,7 @@ export class DestCopy implements IBroccoliPlugin {
 				});
 			}
 		});
-		if (!_.isEmpty(this.dependencies)) {
+		if (!_.isEmpty(this.dependencies) && !this.$config.debugLivesync) {
```

Comment:
I don't agree with this comment. Relations between services in CLI are fact and I don't see how we can avoid them. Conditions are normal part of the development process, we should accept this. If we want to extend and improve the existing livesync feature, we need to add some additional logic. Moving this condition to another place will not make our code better, it will just move the problem to another place. Even worse, it could require many lines of code just to make things working.  If you have better idea that can be applied, I would love to hear it.